resources/views/forum/threads/_form.blade.php

```diff
@@ -1,19 +1,22 @@
-{!! Form::open(['route' => $route, 'method' => $method ?? 'POST']) !!}
+<form action="{{ route($route, $routeParams) }}" method="POST">
```

Comment:
Use argument unpacking instead of the extra variable:

```suggestion
<form action="{{ route(...$route) }}" method="POST">
```